tests/bits/BitVector.cpp

```diff
@@ -259,6 +278,80 @@ TEST_CASE("Test BitVector", "[bits]")
 	REQUIRE(bv_f.count() == 1);
 	bv_f <<= 1;
 	REQUIRE(bv_f.none());
+
```

Comment:
for new tests that we write, let's put each section in its own TEST_CASE

example TEST_CASE("Test Mask_High,Mask_low")